src/core/unicode.js

```diff
@@ -1630,8 +1630,30 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function keepUnifiedIdeograph(str, normalizedUnicodesMap) {
+  // Character maps to a single Unicode; replace it if it is not unified.
+  if (str.length === 1) {
+    if (normalizedUnicodesMap[str] !== undefined) {
+      return normalizedUnicodesMap[str];
+    }
+    return str;
+  }
+  // Character maps to multiple Unicodes; remove ununified Unicodes.
+  let outStr = '';
+  for (let i = 0, ii = str.length; i < ii; i++) {
+    const char = str[i];
+    // If the character does not exist in the map, it's a normal Unicode.
+    if (normalizedUnicodesMap[char] === undefined) {
+      outStr = outStr + char;
+    }
+  }
+  // If outStr==='', return the original str
```

Comment:
This comment isn't necessary, since the code below is self-explanatory.

Also, https://github.com/mozilla/pdf.js/pull/11067#pullrequestreview-276234576 isn't really addressed in any way in the updated patch.

Reply:
What's the difference between my code and your code?
It looks difference. But works the same.

Reply:
The difference is that what I suggested will *only* eliminate outright duplicates (which is the case in your PDF file), whereas your code may remove *arbitrary* unicodes.

These particular examples are probably not realistic/correct, but just to illustrate the principle at hand:
What happens if a `toUnicode` entry contains a character mapped to e.g. `<2f26 2f24>`? Wouldn't you actually want that to be transformed to `<5b50 5927>` (which doesn't happen with this patch)?
Or if a `toUnicode` entry contains e.g. `<2f26 5927>`, would you expect that to be reduced to only `<5927>`?

Again, since the purpose here should be to fix the issue at hand without accidentally regressing something else, it seems that taking some care not the do too much may be helpful.

Reply:
You really think further than me. I will change it later